src/prompt-input/styles.scss

```diff
@@ -161,10 +161,10 @@ $invalid-border-offset: constants.$invalid-control-left-padding;
   @include styles.control-border-radius-full();
 
   &.with-paddings {
-    padding-block-start: styles.$control-padding-vertical;
+    padding-block-start: awsui.$space-scaled-s;
```

Comment:
nit: you can use padding-block and padding-inline instead of padding-block-start, padding-inline-start, ...